wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -139,24 +137,22 @@ namespace :wagn do
       Card::Cache.reset_global
       ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
       prepare_migration
-      paths = ActiveRecord::Migrator.migrations_paths = Card::Migration.migration_paths
+      paths = ActiveRecord::Migrator.migrations_paths = Cardio.migration_paths(:deck_cards)
     
-      Card::Migration.schema_mode do
+      Cardio.schema_mode(:deck_cards) do
         ActiveRecord::Migration.verbose = ENV["VERBOSE"] ? ENV["VERBOSE"] == "true" : true
         ActiveRecord::Migrator.migrate paths, ENV["VERSION"] ? ENV["VERSION"].to_i : nil
       end
     end
```

Comment:
also not necessarily this PR, but it seems like we should be able to do this with less copy/paste